Packs/ArcherRSA/Integrations/ArcherV2/ArcherV2_test.py

```diff
@@ -580,13 +580,40 @@ def test_search_records(self, requests_mock):
         assert records[0]['record']['Id'] == '238756'
         assert records[0]['record']['Device Name'] == 'DEVICE NAME'
 
-    @pytest.mark.parametrize('field_name,field_to_search_by_id,expected_condition', [
-        ('id_field_name', '', '<TextFilterCondition>        <Operator>Contains</Operator>        '
-         + '<Field name="id_field_name">field_id</Field>        <Value>1234</Value></TextFilterCondition >'),
-        ('id_field_name', 'id_field_name', '<ContentFilterCondition>        <Level>5678</Level>        '
-         + '<Operator>Equals</Operator>        <Values><Value>1234</Value></Values></ContentFilterCondition>')
-    ])
-    def test_search_records_soap_request(self, field_name, field_to_search_by_id, expected_condition):
+    @pytest.mark.parametrize(
+        'field_name,field_to_search_by_id,expected_condition',
+        [
+            pytest.param(
```

Comment:
let's explicitly state the keys, since a multiline param is a bit difficult to read
e.g. 
```python
pytest.param(a=a, b=b, ...)
```